doc/rst/source/explain_symbols.rst_

```diff
@@ -1,29 +1,28 @@
 **-S**\ [*symbol*][*size*]
     Plot symbols (including vectors, wedges, fronts, decorated or quoted lines).
-    If present, *size* is symbol size in the unit set in
-    :doc:`gmt.conf` (unless **c**, **i**, or **p** is appended). If the symbol
+    If present, *size* is symbol size in the default unit set by
+    :term:`PROJ_LENGTH_UNIT` (unless **c**, **i**, or **p** is appended). If the symbol
     code (see below) is not given it will be read from the last column in
-    the input data; this cannot be used in conjunction with binary input.
-    Optionally, append **c**, **i**, or
-    **p** to indicate that the size information in the input data is in
-    units of cm, inch, or point, respectively [Default is
+    the input data; this scheme cannot be used in conjunction with binary input.
+    Optionally, append **c**, **i**, or **p** to indicate that the size information
+    in the input data is in units of cm, inch, or point, respectively [Default is
     :term:`PROJ_LENGTH_UNIT`]. **Note**: If you provide *both* size and symbol via the
-    input file you must use :term:`PROJ_LENGTH_UNIT` to indicate the unit
-    used for the symbol size or append the units to the sizes in the file.
-    If symbol sizes are expected via the third data column then you may convert
+    input file you *must* use :term:`PROJ_LENGTH_UNIT` to indicate the unit
+    used for the symbol size or append the units to the sizes listed in the file.
+    If symbol sizes are expected via the a data column then you may convert
     those values to suitable symbol sizes via the **-i** mechanism. The general
     input expectations are:
 
-    *x y* [ *z* ] [ *size* ] [ *symbol-parameters* ] [ *symbol* ]
+    *coordinates* [ *value* ] [ *size* ] [ *symbol-parameters* ] [ *symbol* ]
```

Comment:
Perhaps explain that *coordinates* is *x y* for plot, and *x y z* for plot3d.

Reply:
OK, I did something like that without mentioning modules.